client/apollo/css/src/List/ContentItemDuo/ContentItemDuoCommon.scss

```
@@ -0,0 +1,102 @@
+.af-content-item-duo {
+  display: grid;
+  width: 100%;
+  grid-template:
+    "icon label value"
+    "icon label button";
+  grid-template-columns: auto 1fr 1fr;
+  row-gap: calc(8 / var(--font-size-base) * 1rem);
+
+  &__label,
+  &__value {
+    margin: 0;
+    font-family: var(--font-family-base);
+    font-size: calc(16 / var(--font-size-base) * 1rem);
+    line-height: calc(20 / var(--font-size-base) * 1rem);
```

utilise putôt les valeurs en pourcentage du design (example ci-dessous)
```suggestion
    line-height: 125%;
```